src/aind_behavior_video_transformation/transform_videos.py

```diff
@@ -204,17 +222,16 @@ def convert_video(video_path: Path, dst: Path, arg_set) -> Path:
     output_args = arg_set[1]
 
     ffmpeg_command = ["ffmpeg", "-y", "-v", "warning", "-hide_banner"]
+    # Use 8 threads per compression job
+    ffmpeg_command.extend(["-threads", "8"])
```

Comment:
It might be helpful to make this configurable with 8 set as the default.

Reply:
We should not set this by default... ffmpeg will use all cores by default. Specifying a number less than zero reduces the number of cores each process can use.